src/components/schemas/per_diem_rate.yaml

```diff
@@ -0,0 +1,79 @@
+per_diem_rate_out:
+  type: object
+  additionalProperties: False
+  properties:
+    id:
+      $ref: './fields.yaml#/id_integer'
+    org_id:
+      $ref: ./fields.yaml#/org_id
+    created_at:
+      $ref: ./fields.yaml#/created_at
+    updated_at:
+      $ref: ./fields.yaml#/updated_at
+    is_enabled:
+      $ref: ./fields.yaml#/is_enabled
+    name:
+      type: string
+      maxLength: 255
+      example: Engine
+      description: Currency name
+    description:
+      $ref: ./fields.yaml#/description
+    currency:
+      type: string
+      maxLength: 3
+      example: USD
+      description: Currency code (must be three letters only)
+    rate:
+      type: integer
+      example: 10.0
+      description: Amount per night
+      
+
+per_diem_rate_embed:
+  type: object
+  additionalProperties: False
+  properties:
+    id:
+      $ref: './fields.yaml#/id_integer'
+    name:
+      type: string
+      maxLength: 255
+      example: Engine
+      description: Currency name
+    currency:
+      type: string
+      maxLength: 3
+      example: USD
+      description: Currency code (must be three letters only)
+    rate:
+      type: integer
+      example: 10.0
+      description: Amount per night
+
+
+per_diem_rate_in:
+  type: object
+  additionalProperties: False
+  required: ["name", "currency", "rate"]
```

Comment:
marshmallow schema has is_enabled also as required
either we make it required here or put true as default in marshmallow (same as projects and categories)

Reply:
will add here